### DISCUSSION TITLE
Rust 语言学习历程 - Muxiner's Blog

### DISCUSSION THREAD

**Discussion by giscus** [Muxiner/muxiner.github.io#26] [博客文章评论]
# Rust 语言学习历程 - Muxiner's Blog

无，就是没有。

https://muxiner.github.io/rust-learning/

**Comment by Muxiner** [comment]
# 测试
简简单单发个测试试试。

# 代码

```rust
fn main() {
    println!("World, Hello, Rust, ShaBee.");
}
```

# 图片
**简简单单来个图**：

![](https://raw.githubusercontent.com/Muxiner/BlogImages/main/posts/20221101150411-agnoster.png)
